marketplaceItems/nameTag/appResources/appData/resources/modules/nameTagListManager.js

```diff
@@ -151,14 +136,12 @@ function removeAllLocalEntities(){
 
 
 // Remove a single LocalEntity.
-function removeLocalEntity(uuid, shouldDestory){
+function removeLocalEntity(uuid){
     var avatar = _this.avatars[uuid];
 
-    var type = shouldDestory ? 'destroy' : 'hide';
-
-    avatar.localEntityMain[type]();
+    avatar.localEntityMain.destroy();
     if (avatar.localEntitySub) {
-        avatar.localEntitySub[type]();
+        avatar.localEntityMain.destroy();
```

Comment:
Did you mean to destroy localEntityMain instead of sub here?

Reply:
ooh yeah, I think so, double checking

Reply:
actually I don't need this at all since deleting the main will delete the sub.  good catch